EIPS/eip-4430.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: 4430
+title: Described Transactions
+description: A technique for contract authors to enable wallets to provide a human-readable description of the effect of of a transaction with a given contract.
+author: Richard Moore (@ricmoo), Nick Johnson (@arachnid)
+discussions-to: https://github.com/ethereum/EIPs/issues/4431
```

Comment:
the usual recommendation is to link to a thread in the [Ethereum Magicians](https://ethereum-magicians.org/) Discourse

Reply:
Should I change it? There is already *some* (but not much discussion) on the current link.

Reply:
Yes, all new EIPs should be using Ethereum Magicians. Since this is a new EIP, it should as well.  We currently only make exceptions for existing EIPs (merged as draft before we moved to Ethereum Magicians) that are receiving updates or moving through status changes.